apps/sms/views/shared/js/navigation.js

```diff
@@ -72,11 +71,11 @@ var Navigation = {
     this.transitionPromise = null;
 
     currentPanel = null;
+  },
 
-    Startup.on('post-initialize', function() {
```

Comment:
Removed direct dependency since it's not available in separated view context. But not really happy with ```setReady```....